templates/commands/render.go

```diff
@@ -53,18 +53,34 @@ const (
 
 type Render struct {
 	cli.BaseCommand
+	flags renderFlags
 
 	testFS     renderFS
 	testGetter getterClient
+}
+
+type renderFlags struct {
+	// Positional arguments:
+	source string
+
+	// Flag arguments (--foo):
+	dest           string
+	gitProtocol    string
+	logLevel       string
+	forceOverwrite bool
+	inputs         map[string]string // these are just the --input values from flags; doesn't inclue values from config file or env vars
+	keepTempDirs   bool
+	spec           string
+}
 
-	source             string
-	flagSpec           string
-	flagDest           string
-	flagGitProtocol    string
-	flagLogLevel       string
-	flagForceOverwrite bool
-	flagKeepTempDirs   bool
-	flagInputs         map[string]string
+func (r *renderFlags) forTemplate() map[string]any {
```

Comment:
Since this is for print action only, would it make more sense to let `render_action_print.go` take this responsibility? Let it retrieve values from the flags and build this map.

Reply:
That's a good simplification, thanks. I'll send you a PR shortly.